src/spatialdata/_io/accessor.py

```diff
@@ -0,0 +1,131 @@
+from collections.abc import Callable, Iterator, MutableMapping
+from typing import Any, Literal
+
+import dask.dataframe as dd
+from dask.dataframe.extensions import register_dataframe_accessor, register_series_accessor
+
+
+class _AttrsBase(MutableMapping[str, str | dict[str, str]]):
```

Comment:
We want more flexibility for the values, I'd suggest to do something similar to what `pandas` does: https://github.com/pandas-dev/pandas/blob/46ed6b1c22d78619bb50bc3dee39b9bf718a21d9/pandas/core/generic.py#L321

I propose `MutableMapping[str, Any]`.

Reply:
This is now addressed.